libraries/configuration-editor/source/org/pentaho/reporting/tools/configeditor/util/DOMUtilities.java

```diff
@@ -62,6 +63,8 @@ private DOMUtilities() {
   public static Document parseInputStream( final InputStream instream )
     throws ParserConfigurationException, SAXException, IOException {
     final DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+    dbf.setFeature( XMLConstants.FEATURE_SECURE_PROCESSING, true );
```

Comment:
configuration-editor uses libbase everywhere. I'd rather this whole method went away in favor of using ConfigurationMetaDataParser.parseInputStream and make the libbase one a public static like this one. Then we have one parse thingie in libbase that everyone else can call.